### PR TITLE
advertise storage root error workaround

### PR DESCRIPTION
**Pull request by weichweich**

### CODE REVIEW

File: docs/chain/collator.md [hunk 1]
```diff
@@ -103,6 +103,14 @@ This makes sure that the keyfiles are not accidentally lost or published when th
 You can configure where to store the session keys using the `--keystore-path` option.
 Since the collator will collate only for the parachain, there is no need to add this to the relaychain part of the command.
 
+#### Storage Root Error
+
+Due to an issue in the caching implementation in substrate, it is recommended to reduce the cache size.
+Otherwise the node might get blacklisted by other peers and ultimately disconnected from the p2p network.
+This would prevent a collator from producing blocks.
```
Comment (Dudleyneedham) [inline docs/chain/collator.md]: Eventually, this leads to being banned by the peers until the issue spreads slowing the chain down.

**Review by Dudleyneedham — COMMENTED** [review]
Would like to suggest a minor change.

File: docs/chain/collator.md (hunk 1 above)
Comment (weichweich) [reply docs/chain/collator.md]: like this: 45e8614 ?

**Comment by weichweich** [comment]
LGTD

**Pull request merged**